core/message_manager.py

```
@@ -508,11 +548,21 @@ async def handle_im_message(self, event: KiraMessageEvent):
     async def handle_im_batch_message(self, event: KiraMessageBatchEvent):
         # Start processing
         sid = event.session.sid
+        compression_config = self.kira_config.get_config(
+            "bot_config.image_compression", {}
+        )
 
         for i, message in enumerate(event.messages):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

(B007)